django/db/migrations/state.py

```diff
@@ -229,7 +229,15 @@ def __init__(self, real_apps, models, ignore_swappable=False):
         # mess things up with partial states (due to lack of dependencies)
         self.real_models = []
         for app_label in real_apps:
-            app = global_apps.get_app_config(app_label)
+            try:
+                app = global_apps.get_app_config(app_label)
+            except LookupError as e:
+                new_exception = LookupError(e.args[0] + ' You may be missing this app in your '
+                                            'INSTALLED_APPS setting and have a migration which '
+                                            'depends on it.')
+                new_exception.__cause__ = e
```

Comment:
Can you add an assertion in the test that demonstrates the usefulness of this line?